### ISSUE TITLE
[feature/admin] Automatically add CW on federating instance posts

### ISSUE THREAD

**Issue by tsmethurst** [superseriousbusiness/gotosocial#9]
From a post by a Mastodon user who would rather remain anonymous:

'damn it'd be really cool if i could somehow make my instance about cooking automatically add a "food" cw when posts federate out, but not show the cw on the instance itself, this sounds possible but i have no idea how to do it'

also:

'as far as glitch-soc implements it, an existing CW will overwrite the automatic one. the automatic one only applies to posts that don't have a CW'

GoToSocial should be able to do this, it's a really cool idea.

**Comment by tsmethurst** [comment]
> A instance should be able to add a CW on incoming federated posts too. Like, add a NSFW CW on notes comming from Switter.
> 
> Pleroma do thin with [MRF](https://git.pleroma.social/pleroma/pleroma/-/blob/develop/docs/configuration/mrf.md), Message Rewrite Facility. A rule based message rewrite for incoming and outgoing activities adds a lot of flexibility and moderation power.

Good point, yes! I will definitely add that as well :) Thanks for the suggestion

**Comment by NyaaaWhatsUpDoc** [comment]
see: https://github.com/superseriousbusiness/gotosocial/issues/835#issuecomment-1959202929

**Comment by daenney** [comment]
We've migrated to Codeberg. Your issue has been imported as https://codeberg.org/superseriousbusiness/gotosocial/issues/9.
